edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.Core.Test/ServiceIdentityTreeTest.cs

```diff
@@ -55,7 +55,7 @@ internal static ServiceIdentity CreateServiceIdentity(string deviceId, string mo
                 "1234",
                 capabilities,
                 new ServiceAuthentication(ServiceAuthenticationType.None),
-                ServiceIdentityStatus.Enabled);
+                isEnabled ? ServiceIdentityStatus.Enabled : ServiceIdentityStatus.Disabled);
```

Comment:
ServiceIdentityStatus only contains Enabled and Disabled enum values.  Do you know why we use enum type for ServiceIdentityStatus, but not boolean?

Reply:
I don't know the history for sure, but if I have to guess I'd imagine it's because we mirror Device class from the SDK, which also uses an enum instead of a boolean.

---
In reply to: [458523533](https://github.com/Azure/iotedge/pull/3248#discussion_r458523533) [](ancestors = 458523533)